src/collective/cover/tiles/file.py

```diff
@@ -79,10 +79,19 @@ def download_widget(self):
         obj = uuidToObject(self.data['uuid'])
         if obj:
             url = obj.absolute_url()
-            icon = obj.getBestIcon()
             portal_url = obj.portal_url()
-            mime = obj.lookupMime(obj.getField('file').getContentType(obj))
-            size = obj.get_size()
+
+            try:
```

Comment:
@pbauer @thet this is the kind of things I'm stuck with; I need your wisdom on the best way to implement this and/or fix plone.app.contenttypes